### PR TITLE
[DebuggerV2] Clicking tensor in graph execution component brings up graph view

### PR DESCRIPTION
**Pull request by caisq**
* Motivation for features / changes
  * Continue developing the graph-related frontend of DebuggerV2
* Technical description of changes
  * Add a click event listener to each tensor name in GraphExecutionComponent
  * Change the tensor name element from a `<div>` to a `<button>` to reflect the
     fact that it is clickable.
  * The listener dispatches the `graphOpFocused()` action with the corresponding
    `graph_id` and `op_name`, so the the `GraphComponent` can bring up the
    local graph structure view of the op.
  * Button outline CSS in GraphOpComponent to avoid extraneous outlines when
    focused.
* Screenshots of UI changes
  * image
* Detailed steps to verify changes work correctly (as executed by you)
  * Unit test added
  * Manual testing against real logdirs with tfdbg2 data (see screenshot above)

### CODE REVIEW

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph_executions/graph_executions_component.css [hunk 1]
```diff
@@ -67,24 +67,34 @@ limitations under the License.
...
 .tensor-name {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
   direction: rtl;
   display: block;
   height: 16px;
   line-height: 16px;
+  margin: 2px 0 1px;
+  max-width: calc(100% - 3px);
```
Comment (stephanwlee) [inline tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph_executions/graph_executions_component.css:87]: `box-sizing: border-box;` tho :(

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph_executions/graph_executions_container_test.ts [hunk 2]
```diff
@@ -148,6 +151,29 @@ describe('Graph Executions Container', () => {
       By.css('debug-tensor-value')
     );
     expect(debugTensorValueElements.length).toBe(tensorContainers.length);
+
+    // Clicking the tensor name buttons should dispatch the proper action.
```
Comment (stephanwlee) [inline tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph_executions/graph_executions_container_test.ts:155]: sounds like a name of a new behavior test spec.

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph_executions/graph_executions_component.css (hunk 1 above)
Comment (caisq) [reply tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph_executions/graph_executions_component.css]: Good point. Using `box-sizing: border-box;` is more appropriate here. Adde it.

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph_executions/graph_executions_container_test.ts (hunk 2 above)
Comment (caisq) [reply tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph_executions/graph_executions_container_test.ts]: Done making this a standalone `it()`.

**Pull request merged**